src/main/java/org/gridsuite/network/map/model/ElementType.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.network.map.model;
+
+import com.powsybl.iidm.network.*;
+import org.gridsuite.network.map.dto.ElementInfos;
+import org.gridsuite.network.map.dto.hvdc.AbstractHvdcInfos;
+import org.gridsuite.network.map.dto.line.AbstractLineInfos;
+import org.gridsuite.network.map.dto.load.AbstractLoadInfos;
+import org.gridsuite.network.map.dto.substation.AbstractSubstationInfos;
+import org.gridsuite.network.map.dto.threewindingstransformer.AbstractThreeWindingsTransformerInfos;
+import org.gridsuite.network.map.dto.twowindingstransformer.AbstractTwoWindingsTransformerInfos;
+import org.gridsuite.network.map.dto.voltagelevel.AbstractVoltageLevelInfos;
+
+import java.util.function.BiFunction;
+
+
+public enum ElementType {
```

Comment:
It shouldn't be "NetworkElementType" ? 
Element is even worse than equipment no ?

Reply:
answer : we're already in network map server so no need to specify

Reply:
In this micro service `element` means `network element` because these are the only objects managed by it